modules/openapi-generator/src/main/resources/python-fastapi/setup_cfg.mustache

```diff
@@ -11,13 +11,10 @@ classifiers =
     Programming Language :: Python :: {{{generatorLanguageVersion}}}
 
 [options]
-install_requires =
-    fastapi[all]
-setup_requires =
-    setuptools
-package_dir=
-    =src
-packages=find_namespace:
+install_requires = fastapi[all]
+setup_requires = setuptools
+package_dir = ={{sourceFolder}}
```

Comment:
Should this have a double equals here?

Reply:
According to https://setuptools.pypa.io/en/latest/userguide/declarative_config.html?highlight=package_dir#using-a-src-layout this option needs double equals.